### PR TITLE
Release 2309 deployment context

### PR DESCRIPTION
**Pull request by ehsanb29**
Create new deployment context profiles for the current release(23.09)
Update GLB to API timeout.

### CODE REVIEW

File: profiles/deployment_context.dev-ppp [hunk 1]
```diff
@@ -7,32 +7,32 @@ config_project_id         = "open-targets-partner-platform"
...
+config_openai_api_docker_image_version = "0.0.10"
 
 // --- DNS configuration --- //
 config_dns_platform_subdomain     = "partner-platform"
 config_dns_subdomain_prefix       = "dev"
 config_dns_platform_api_subdomain = "api"
 
 // --- Web App configuration --- //
-config_webapp_release = "v0.4.11"
+config_webapp_release = "v0.5.11"
```
Comment (mbdebian) [inline profiles/deployment_context.dev-ppp:33]: I think we have 0.5.12 in PPP production, would you mind making sure that our development environments match production after a release?
Thanks!

**Review by mbdebian — CHANGES_REQUESTED** [review]
Thanks for the PR @ehsanb29 !
Would you mind having a look at my comments?
Cheers,
Manuel

File: profiles/deployment_context.dev-ppp (hunk 1 above)
Comment (ehsanb29) [reply profiles/deployment_context.dev-ppp:33]: Yes, The latest webapp version(0.5.12) has been created after the production deployment(PPP and Platform) and I didn't deploy it to Dev yet.
I'll do it now on. Thanks for the comment.

**Review by mbdebian — APPROVED** [review]
Thanks for the PR @ehsanb29 !
I think this is ready for merge.
Please, double check the merge process with @jdhayhurst 
Thanks !

**Pull request merged**